Documentation/botr/readytorun-format.md

```diff
@@ -483,9 +597,12 @@ enum ReadyToRunHelper
 
     // JIT32 x86-specific exception handling
     READYTORUN_HELPER_EndCatch                  = 0x110,
+
+    // A flag to indicate that a helper call uses VSD
+    READYTORUN_HELPER_FLAG_VSD                  = 0x10000000,
```

Comment:
This internal crossgen implementation hack. It is not part of the file format spec. Correct? This document should be the file format spec, insulated from the implementation hacks.

Reply:
OK, will remove that. In fact I originally ported this to CPAOT as I was under the impression that it actually is a part of the format but upon closer scrutiny I concur it's just an internal contract within zapinfo. I'll take a look whether I can easily remove it from CPAOT too.

Reply:
Removed in 6th commit.

Reply:
I would be nice to delete it from CPAOT and R2R dump too.

Reply:
Sure, will do.